cg/test/processors/processors.api.spec.js

```diff
@@ -58,14 +58,17 @@ describe('Processor requesting data', () => {
   });
 
   it('should respond with the decrypted data for the subject requested', async () => {
+    // Given
     const key = encryption.generateClientKey();
     const testData = { test: 'true' };
     const subjectId = hash('p1');
     await db('subjects').insert({
```

Comment:
it would be valuable to create a helper that sets up the subject for you at this point, lots of setup code could be removed.